client/client.go

```diff
@@ -266,6 +289,7 @@ func MakeConfiguration(file string) (err error) {
 	}
 	fmt.Fprintf(fd, "[api]\n\turl = \"%s\"\n", cfg.API.URL)
 	fmt.Fprintf(fd, "[gpg]\n\thome = \"%s\"\n\tkeyid = \"%s\"\n", cfg.GPG.Home, cfg.GPG.KeyID)
+	fmt.Fprintf(fd, "[TerminalConf]\n\tcolor = off\n")
```

Comment:
But the above method is so lengthy. Sorry for your time. I want to write code which work as
```
if (flag == off )
  remove (\x1b[__;_m and \x1b[0m)
```
from entire code before building binary. Currently i have no idea how to do that but i will do.
any suggestion ?